packages/agreement-process/test/utils.ts

```diff
@@ -50,7 +51,7 @@ export const addOneAgreement = async (
   agreements: AgreementCollection
 ): Promise<void> => {
   await writeAgreementInEventstore(agreement, postgresDB);
-  await writeInReadmodel(agreement, agreements);
+  await writeInReadmodel(toReadModelAgreement(agreement), agreements);
```

Comment:
Could you do the same in other test utils files where we write agreements? We do the same also in `tenant` and `catalog` test utils for example (maybe also attribute?)